app/locales/bg.json

```diff
@@ -33,7 +33,8 @@
     "microbreak": "микропочивка",
     "break": "почивка",
     "nextBreakFollowing": "\nСледващата почивка ще бъде след {{count}} микропочивка",
-    "nextBreakFollowing_plural": "\nСледващата почивка ще бъде след {{count}} микропочивки"
+    "nextBreakFollowing_plural": "\nСледващата почивка ще бъде след {{count}} микропочивки",
+    "notificationStatus": "In do not disturb mode"
```

Comment:
if you check the tray message, you will notice that `In do not disturb mode` is not on right position (it stays in first line: `stretchly - break time reminder appIn do not disturb mode`). You can new line with `\n`

Reply:
Or maybe add new line in code where we create tooltip message?